src/Tribe/Main.php

```diff
@@ -225,7 +225,7 @@ public function load_assets() {
 			$this,
 			[
 				[ 'tribe-accessibility-css', 'accessibility.css' ],
-				[ 'tribe-query-string', 'utils/query-string.js' ],
+				[ 'tribe-query-string', 'utils/query-string.js', null, null, [ 'in_footer' => [ 'strategy' => 'defer' ] ] ],
```

Comment:
Why are we changing this to defer? Is that fixing a particular bug?

Reply:
I believe it was just to optimize the page load. @adamsilverstein ?